index.bs

```diff
@@ -5521,6 +5521,17 @@ Methods</h4>
 		</pre>
 </dl>
 
+<h4 id="listenerprocessing">
+Processing</h4>
+
+Since the {{AudioListener}} has {{AudioParam}}, and {{AudioNode}} can be
+connected to {{AudioParam}}, it is necessary to have the {{AudioListener}}
+participate in the {{AudioNode}} ordering algorithm.
```

Comment:
How about something like this? [5527-5534]

Because AudioListener's parameters can be connected with AudioNodes and also they affect the output of PannerNodes in the same graph, the node ordering algorithm should take the AudioListener into consideration when computing the order of processing. For this reason, all the PannerNodes in the graph have the AudioListener as input.